charts/gateway-helm/values.tmpl.yaml

```diff
@@ -74,6 +74,15 @@ deployment:
 service:
   annotations: {}
 
+hpa:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 1
+  targetCPU: 50
```

Comment:
thinking out loud, should there be a percentage suffix here ?

Reply:
Well manifests don't have/required it. I imagine you could, but it would be ineffectual.

Anyway, I think this will be removed in favour of: https://github.com/envoyproxy/gateway/pull/5127#discussion_r1927272320

Reply:
sure lets go with below comment